develop/application-devs/interact/understand-parachain-data.md

```diff
@@ -0,0 +1,170 @@
+
+For example, the following is a condensed excerpt of the calls for the Sudo pallet:
+
+```json
+--8<-- 'code/application-devs/interact/sudo-metadata-calls.json'
+```
+
+For each field, you can access type information and metadata for the following:
+
+- **Storage metadata** - provides the information required to enable applications to get information for specific storage items
+- **Call metadata** - includes information about the runtime calls defined by the `#[pallet]` macro including call names, arguments and documentation
+- **Event metadata** - provides the metadata generated by the `#[pallet::event]` macro, including the name, arguments, and documentation for each pallet event
+- **Constants metadata** - provides metadata generated by the `#[pallet::constant]` macro, including the name, type, and hex-encoded value of the constant
+- **Error metadata** - provides metadata generated by the `#[pallet::error]` macro, including the name and documentation for each pallet error
+
+!!!note
+    Type identifiers change from time to time so you should avoid relying on specific type identifiers in your applications
+
+### Extrinsic
+
+Extrinsic metadata is generated by the runtime and provides useful information about how transactions are formatted. When decoded, the metadata contains the transaction version and the list of signed extensions.
+
+For example:
+
+```json
+--8<-- 'code/application-devs/interact/extrinsic-metadata.json'
+```
+
+The type system is [composite](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/frame_runtime_types/index.html){target=\_blank}, meaning each type identifier contains a reference to a specific type or to another type identifier that provides information about the associated primitive types.
+
+For example, you can encode the `BitVec<Order, Store>` type, but to decode it properly you must know the types used for the `Order` and `Store` types. To find type information for `Order` and `Store`, you can use the path in the decoded JSON to locate their type identifiers.
+
+## Other RPC APIs
+
+A standard node comes with the following APIs to interact with a node:
+
+- [`AuthorApiServer`](https://paritytech.github.io/polkadot-sdk/master/sc_rpc/author/trait.AuthorApiServer.html){target=\_blank} - make calls into a full node, including authoring extrinsics and verifying session keys
+- [`ChainApiServer`](https://paritytech.github.io/polkadot-sdk/master/sc_rpc/chain/trait.ChainApiServer.html){target=\_blank}  - retrieve block header and finality information
+- [`OffchainApiServer`](https://paritytech.github.io/polkadot-sdk/master/sc_rpc/offchain/trait.OffchainApiServer.html){target=\_blank}  - make RPC calls for offchain workers
+- [`StateApiServer`](https://paritytech.github.io/polkadot-sdk/master/sc_rpc/state/trait.StateApiServer.html){target=\_blank}  - query information about on-chain state such as runtime version, storage items, and proofs
```

Comment:
```suggestion
- [`StateApiServer`](https://paritytech.github.io/polkadot-sdk/master/sc_rpc/state/trait.StateApiServer.html){target=\_blank} - query information about on-chain state such as runtime version, storage items, and proofs
```

Reply:
Sorry, I accidentally added in extra spaces. This is what I get for doing it on GH browser